src/jasmineTestRunner/JasmineReporter.js

```diff
@@ -25,7 +25,7 @@ function JasmineReporter(config) {
   jasmine.Reporter.call(this);
   this._config = config || {};
   this._logs = [];
-  this._resultsDeferred = Promise.defer();
+  this._resultsPromise = new Promise((resolve) => { this._resolve = resolve; });
```

Comment:
not really. It should be `resolve => this._resolve = resolve` :P Merged it anyway #yolo